test/SCRStub.rb

```diff
@@ -58,3 +58,25 @@ def stub_scr_read(path_name)
     allow(Yast::SCR).to receive(:Read).with(path).and_return info
   end
 end
+
+module NetworkInterfacesStub
```

Comment:
this is a bit misleading name. NetworkInterfaces is a module ... and you stub not only this module inside